tests/firedrake_adjoint/test_dynamic_meshes.py

```diff
@@ -1,12 +1,14 @@
 import pytest
 from firedrake import *
 from firedrake_adjoint import *
+import ufl
```

Comment:
`FiniteElement` is already imported through firedrake.

Reply:
fixed now